daemon/cmd/daemon.go

```diff
@@ -425,6 +428,17 @@ func NewDaemon(ctx context.Context, epMgr *endpointmanager.EndpointManager, dp d
 	}
 	d.policyTrigger = t
 
+	regenerationTrigger, err := trigger.NewTrigger(trigger.Parameters{
+		Name:            "datapath_regeneration",
```

Comment:
```suggestion
		Name:            "datapath-regeneration",
```

There's a pattern in the code to use dashes.

Reply:
Ah seems I followed the pattern of `policy_update`
```
./pkg/ipam/node_manager.go-             Name:            "ipam-node-manager-resync",
./pkg/ipam/node_manager.go-                     Name:            fmt.Sprintf("ipam-pool-maintainer-%s", resource.Name),
./pkg/ipam/node_manager.go-                     Name:        fmt.Sprintf("ipam-pool-maintainer-%s-retry", resource.Name),
./pkg/ipam/node_manager.go-                     Name:            fmt.Sprintf("ipam-node-k8s-sync-%s", resource.Name),
./pkg/ipam/crd.go-              Name:        "crd-allocator-node-refresher",
./pkg/endpoint/endpoint.go-                     Name:        "sync_endpoint_header_file",
./daemon/cmd/daemon.go-         Name:            "policy_update",
```